packages/utils/src/fallbackSubgraph.ts

```diff
@@ -180,6 +184,10 @@ export class FallbackSubgraph<T extends SubgraphSdk> {
    * @returns Subgraph sync records for each subgraph.
    */
   public async sync(_latestBlock?: number): Promise<SubgraphSyncRecord[]> {
+    // If the latest sync was within 5s, do not requery
+    if (Date.now() - this.latestSync < 5_000) {
```

Comment:
make this a const at the top? And I feel like this should be higher. Like, half a quarter of our max lag value (which is in blocks, so hmmm) or something...

Would be cool if we could pass in like, 1/4 * maxLag * blockLengthMs (and get the blockLengthMs from chain reader, which determines that value on init). If no time for that, can we leave a TODO?

Reply:
will leave as a todo and make a const